lake/modules/transpose_buffer_aggregation.py

```diff
@@ -142,5 +142,5 @@ def send_tba_rdy(self):
 
 
 if __name__ == "__main__":
-    dut = TransposeBufferAggregation(1, 4, 3, 3, 5, 2)
-    verilog(dut, filename="tba.sv")
+    dut = TransposeBufferAggregation(1, 4, 3, 1, 5, 2, 2)
```

Comment:
Can use kwargs so that the reader will know what each argument means? For now it's just random numbers. Same thing applies to other constructor call.

Reply:
Sounds good.